user-guide/Advanced_Functionality/Failover/Ending_a_Failover_configuration.md

```diff
@@ -6,7 +6,32 @@ uid: Ending_a_Failover_configuration
 
 When two DMAs are linked together in a Failover setup, it is possible to end the Failover configuration, and thereby separate the two DMAs again.
 
-To do so:
+> [!CAUTION]
+> While it is possible from DataMiner 10.2.0 [CU9]/10.2.12 onwards to decommission a Failover setup while the server hosting the offline Agent is unavailable, this will not reset the offline Agent to a clean installation. As such, you will need to make sure the offline Agent is not brought online again in the network as this would cause double polling. In addition, there could be minor data loss and inconsistencies on the database level as the database node cannot be cleanly decommissioned.
+
+## Verifying the replication factor of the Cassandra database
+
+In case you use local Cassandra it is best to verify the replication factor of your Cassandra configuration. As breaking up the Failover configuration  will revert the database to single node, you need to make sure that the data will be included on both nodes. this means that the replication factor must be set to **2**.
```

Comment:
What do you mean with "if you use local Cassandra"? A setup with a Cassandra database per DMA, rather than a Cassandra Cluster setup?

Reply:
@MariekeGO yes correct :)